data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/damage_type/damage_type_applied_to_both.cfg

```diff
@@ -0,0 +1,80 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: apply_to=both with [filter_self][filter_weapon]type= and [filter_opponent][filter_weapon]type=
+##
+# Actions:
+# Alice and Bob are both of type Test Melee Quintain.
+# change type of attack of pierce and resistance to -50%.
+# Give Alice's weapon specials [damage_type]applied to both active if alice and bob attack are type=pierce.
+# Have Alice attack with his weapon.
+##
+# Expected end state:
+# Deterministic end state, without crashing, but BROKE STRICT.
+# Damage_type is active
+# Bob takes 15 damage.
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "damage_type_applied_to_both_with_both_filtering" (
```

Comment:
Change the test name to "damage_type_apply_to_both_filter_self".

Reply:
done

Reply:
It's missing the `_opponent` part.

Reply:
fixed